src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -100,4 +100,33 @@ public FunctionsController(WebScriptHostManager scriptHostManager, WebHookReceiv
             return response;
         }
     }
+    
+    // Custom web hooks for extensions. 
+    public class HookController : ApiController
```

Comment:
Both O365 (Graph webhooks) and DTF extensions need this.  The customer facing impact of doing BYOB is to enable new extensions like those. 

---
In reply to: [108998785](https://github.com/Azure/azure-webjobs-sdk-script/pull/1349#discussion_r108998785) [](ancestors = 108998785)